### DISCUSSION TITLE
How can you confirm that HA is enabled and working properly?

### DISCUSSION THREAD

**Discussion by danfinn** [grafana/grafana#66613] [Alerting]
I've enabled HA for alerting using the helm chart values.  I don't have any errors in my logs but I also don't see any way to confirm that:

1) HA is enabled and working
2) see which nodes are setup for HA

**Comment by danfinn** [comment]
This is about as much information as I can gather from my logs:

logger=ngalert.multiorg.alertmanager component=cluster t=2023-04-14T19:08:41.956939364Z level=info msg="gossip settled; proceeding" elapsed=10.003069235s

**Reply by JohnnyQQQQ** [reply]
Grafana provides a selection of metrics that can be employed to monitor various aspects, such as the health of the Alertmanager cluster (`alertmanager_cluster_health_score`). To enable these metrics, you will need to adjust the configuration settings. Once enabled, the metrics can be accessed via the `/metrics` endpoint. 

For more information on configuring Grafana monitoring, please visit the following link: https://grafana.com/docs/grafana/latest/setup-grafana/set-up-grafana-monitoring/

**Comment by danfinn** [comment]
It would be helpful to add this to the alerting docs, right now there is no mention of it at all related to alerting that I could find.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This looks to be answered, so I will close the discussion. Feel free to make an [issue](https://github.com/grafana/grafana/issues) asking for docs updates if you feel they are still needed.
